packages/js/settings-editor/src/legacy/content.tsx

```diff
@@ -27,20 +27,20 @@ export const LegacyContent = ( {
 	return (
 		<form>
 			<div className="woocommerce-settings-content">
-				{ section.settings.map( ( data ) => {
-					const key =
-						data.id +
-						'-' +
-						( data.title ?? '' ).replace( /\s+/g, '-' ) +
-						'-' +
-						data.type;
```

Comment:
`CustomSettingsField` doesn't have `id` and `title` properties so I use type + `key` as the unique identifier.